lib/schema.js

```diff
@@ -74,9 +74,13 @@ function getSchemaPrefix(params, timestamp) {
 */
 function generateKey(params, metric, timestamp) {
     const prefix = getSchemaPrefix(params, timestamp);
+    if (params.location) {
```

Comment:
Is this special case needed, what about simply adding the `locationStorage` key to the global `keys` object? It looks like what you added to support the location property in `getSchemaPrefix()` was meant for this.

Reply:
I agree that we should maintain the common keys objects, perhaps updating one of the "state keys" (or at least move this logic in `getSchemaPrefix`. My feeling is that the metrics should be exposed through an API listing as are other metrics that Utapi is tracking.